app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -32,6 +33,13 @@ public function panel(Panel $panel): Panel
             ])
             ->discoverResources(in: app_path('Filament/Admin/Resources'), for: 'App\\Filament\\Admin\\Resources')
             ->discoverPages(in: app_path('Filament/Admin/Pages'), for: 'App\\Filament\\Admin\\Pages')
-            ->discoverWidgets(in: app_path('Filament/Admin/Widgets'), for: 'App\\Filament\\Admin\\Widgets');
+            ->discoverWidgets(in: app_path('Filament/Admin/Widgets'), for: 'App\\Filament\\Admin\\Widgets')
+            ->plugins([
+                FilamentLogViewer::make()
+                    ->authorize(fn () => user()->can('view panellog'))
```

Comment:
```suggestion
                    ->authorize(fn () => user()->can('view panelLog'))
```

Reply:
Upper works on your machine, does not work on my machine.... Soooo what do we do?